src/components/custom-field-input-text/custom-field-input-text.jsx

```diff
@@ -44,9 +54,11 @@ CustomFieldInputText.propTypes = {
   helpText: PropTypes.string,
   id: PropTypes.string,
   name: PropTypes.string,
+  onChange: PropTypes.func,
   onClick: PropTypes.func,
   placeholder: PropTypes.string,
   required: PropTypes.bool,
+  type: PropTypes.string,
```

Comment:
If we don't drop this, we will want to restrict the values to a limited set. I've seen misuses of `type` when we were intending a text-like field as a checkbox :(

Reply:
Do we want to work with @annemnichols to define the valid set of text?

Reply:
@juanca I'm just now realizing I mis-interpreted this statement, and I definitely agree. For these inputs, we can probably just do `date||number||text`, does that sound okay?

Reply:
Yeah, at the very least those are text-like. :+1: